README.md

```diff
@@ -33,7 +33,7 @@ We also provide reusable GO packages.
 
 ## Applications
 
-- [Golang REST API boilerplate](https://github.com/nanlabs/nancy.go/tree/main/examples/golang-todo-rest-crud/) - REST API to create, update and retrieve Entities, including grateful shutdown, rate limiting, structured logging, unit tests, integration tests, environment variables, health check and API documentation with swagger. Technologies: Golang 1.19, MongoDB (with Docker Compose), Gorilla Mux, Go Swagger, Tollbooth (rate limiting), Zap (logging), Viper, Mockery, Makefile, Pre-commit, and DockerTest (integration tests). _Keywords: Golang, REST API, MongoDB, Gorilla Mux, Go Swagger, Tollbooth, Zap, Viper, Mockery, Makefile, Pre-commit, Docker, Docker Compose, DockerTest_
+- [Golang REST API boilerplate](https://github.com/nanlabs/nancy.go/tree/main/examples/golang-todo-rest-crud/) - REST API to create, update and retrieve Entities, including graceful shutdown, rate limiting, structured logging, unit tests, integration tests, environment variables, health check and API documentation with swagger. Technologies: Golang 1.19, MongoDB (with Docker Compose), Gorilla Mux, Go Swagger, Tollbooth (rate limiting), Zap (logging), Viper, Mockery, Makefile, Pre-commit, and DockerTest (integration tests). _Keywords: Golang, REST API, MongoDB, Gorilla Mux, Go Swagger, Tollbooth, Zap, Viper, Mockery, Makefile, Pre-commit, Docker, Docker Compose, DockerTest_
```

Comment:
we need to update the file `examples.json` instead of this one 👌🏻 the README is autogenerated 👌🏻